imports/ui/containers/Preset/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import MedContainer from '../../components/MediumContainer';
+import styles from './styles.css';
+
+import PresetSelection from './PresetSelection';
+import CategoryButton from '../../components/CategoryButton';
+
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  Link
+} from 'react-router-dom'
+
+class Preset extends Component {
```

Comment:
'Preset' is not a meaningful enough component name. Component names should signal something about what the component does, or is. Consider the reusability of all of your components and try to be as generic as possible.